tests/integration/test_counts.py

```diff
@@ -0,0 +1,115 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from tests.integration import test_submit, utility_test_functions
+
+workflows_path = Path(__file__).parents[1].joinpath("workflows/")
+
+
+def check_status(test_workflow_id: str):
```

Comment:
Should this function be renamed to better reflect actual usage, i.e. wait for the workflow to reach desired status (succeed or fail in this case)?

Reply:
done